js-miniapp-sdk/package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Mini App SDK for JavaScript",
   "main": "build/src/index.js",
   "types": "build/src/index.d.ts",
-  "dependencies": {},
+  "dependencies": {
+    "browserify": "^16.5.1"
```

Comment:
This one isn't being used yet, right? Maybe remove the dependency for now and we can add it back in if we use browserify later.